internal/persistence/inmemory/inmemory_test.go

```diff
@@ -19,9 +19,9 @@ import (
 	"strings"
 	"testing"
 
+	_ "github.com/mattn/go-sqlite3" // Load drivers for sqlite3
```

Comment:
Unrelated to the lint fixes, but since this line got moved: do we need the sqlite drivers linking in here?

Reply:
No, it's gone now.